source/PreferencesPanel.cpp

```diff
@@ -599,6 +602,11 @@ void PreferencesPanel::DrawSettings()
 			isOn = true;
 			text = Preferences::BoardingSetting();
 		}
+		else if(setting == BACKGROUND_PARALLAX)
+		{
+			isOn = true;
```

Comment:
This will make the preference always on, which it shouldn't be if it's on "off".

Reply:
The default is the value you assigned to `parallaxIndex` at the top. This is to control whether the preference entry is highlighted or not.